lib/pubsub/iam.js

```diff
@@ -0,0 +1,190 @@
+/*!
+ * Copyright 2014 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*!
+ * @module pubsub/iam
+ */
+
+'use strict';
+
+var is = require('is');
+var arrify = require('arrify');
+
+/*! Developer Documentation
+ *
+ * @param {module:pubsub} pubsub - PubSub Object
+ * @param {string} resource - topic or subscription name
+ */
+/**
+ * IAM (Identity and Access Management) policies are used to create user groups
+ * and designate roles to your project members. Using these groups and roles you
+ * can grant or restrict access to any of your resources.
+ *
+ * @constructor
+ * @alias module:pubsub/iam
+ */
+function IAM(pubsub, resource) {
+  this.resource = resource;
+  this.makeReq_ = pubsub.makeReq_.bind(pubsub);
+}
+
+/**
+ * Get the IAM Policy
+ *
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request.
+ * @param {object} callback.policy - The policy.
+ * @param {object} callback.apiResponse - The full API response.
+ *
+ * @alias iam.getPolicy
+ *
+ * @example
+ * var pubsub = gcloud.pubsub({
+ *   projectId: 'grape-spaceship-123'
+ * });
```

Comment:
Can't believe I switched Topic and Subscription :S I've been doing that a lot lately!

Can you just add `keyFilename: '/path/to/keyfile.json'` below `projectId: ...` to be consistent with our other constructor docs?